scss/components/_buttons.scss

```diff
@@ -407,9 +404,25 @@ $btn-border-width: 4px;
     }
   }
 }
-
 @media screen and (-ms-high-contrast: active) {
   .btn {
     border: $btn-border-width solid;
+    color: windowText !important;
+  }
+
+  a.btn {
+    -ms-high-contrast-adjust: none;
```

Comment:
I understand now.  Without this line, do these styles work in IE and legacy Edge (EdgeHTML, not Chromium)?

Reply:
Without this line it works in IE , i dont have access to EdgeHTML (Its not available in browserstack also). I have made 1 adjustment for IE in my latest commit